conda-store-server/tests/conftest.py

```diff
@@ -14,19 +14,26 @@
 @pytest.fixture
 def celery_config(conda_store):
     config = conda_store.celery_config
-    config["traitlets"] = {"CondaStore": {"database_url": conda_store.database_url}}
+    config["traitlets"] = {"CondaStore": {
+        "database_url": conda_store.database_url,
+        "store_directory": conda_store.store_directory,
+    }}
     return config
 
 
 @pytest.fixture
 def conda_store_config(tmp_path, request):
     from traitlets.config import Config
 
-    filename = pathlib.Path(tmp_path) / "database.sqlite"
+    filename = tmp_path / ".conda-store" / "database.sqlite"
+
+    store_directory = tmp_path / ".conda-store" / "state"
+    store_directory.mkdir(parents=True)
 
     with utils.chdir(tmp_path):
         yield Config(
             CondaStore=dict(
+                store_directory=str(store_directory),
                 database_url=f"sqlite:///{filename}?check_same_thread=False"
```

Comment:
This needs to be rebased on the Windows PR, to make sure these work on the Windows builder.